### PR TITLE
manual updates 20220531 dependencies

### PR DESCRIPTION
**Pull request by moljac**
### Describe your contribution

- com.google.android.datatransport:transport-backend-cct - 3.1.4 -> 3.1.5
- com.google.android.datatransport:transport-runtime - 3.1.4 -> 3.1.5
- com.google.dagger:dagger - 2.41 -> 2.42
- com.google.errorprone:error_prone_annotations - 2.10.0 -> 2.7.1
- com.google.protobuf:protobuf-javalite - 3.19.2 -> 3.17.3
- io.grpc:grpc-android - 1.45.1 -> 1.43.2
- io.grpc:grpc-api - 1.45.1 -> 1.43.2
- io.grpc:grpc-context - 1.45.1 -> 1.43.2
- io.grpc:grpc-core - 1.45.1 -> 1.43.2
- io.grpc:grpc-okhttp - 1.45.1 -> 1.43.2
- io.grpc:grpc-protobuf-lite - 1.45.1 -> 1.43.2
- io.grpc:grpc-stub - 1.45.1 -> 1.43.2
- io.opencensus:opencensus-api - 0.29.0 -> 0.30.0
- io.opencensus:opencensus-contrib-grpc-metrics - 0.29.0 -> 0.30.0
- io.perfmark:perfmark-api - 0.24.0 -> 0.25.0
- org.chromium.net:cronet-api - 92.4515.131 -> 94.4606.61
- org.chromium.net:cronet-common - 92.4515.131 -> 94.4606.61
- org.chromium.net:cronet-embedded - 92.4515.131 -> 94.4606.61
- org.chromium.net:cronet-fallback - 92.4515.131 -> 94.4606.61
- org.codehaus.mojo:animalsniffer-annotations - 1.19 -> 1.20
- org.tensorflow:tensorflow-lite - 2.6.0 -> 2.7.0
- org.tensorflow:tensorflow-lite-api -  -> 2.7.0
- org.tensorflow:tensorflow-lite-gpu - 2.6.0 -> 2.7.0
- com.google.errorprone:error_prone_annotations -  -> 2.9.0
- com.google.protobuf:protobuf-java-lite -  -> 3.19.2

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1353,17 +1353,17 @@
...
         "nugetId": "Xamarin.Google.Dagger",
         "dependencyOnly": false,
         "templateSet": "dagger"
       },
       {
         "groupId": "com.google.errorprone",
         "artifactId": "error_prone_annotations",
-        "version": "2.10.0",
-        "nugetVersion": "2.10.0",
+        "version": "2.7.1",
```
Comment (jpobst) [inline config.json:1365]: Many of these dependencies are going backwards.
- Xamarin.Google.ErrorProne.Annotations
- Xamarin.Protobuf.JavaLite
- Xamarin.Grpc.Android

File: config.json [hunk 2]
```diff
@@ -1564,62 +1564,71 @@
...
         "nugetId": "Xamarin.TensorFlow.Lite",
         "dependencyOnly": false,
         "templateSet": "tensorflow-lite"
       },
+      {
+        "groupId": "org.tensorflow",
+        "artifactId": "tensorflow-lite-api",
+        "version": "2.7.0",
+        "nugetVersion": "2.7.0",
+        "nugetId": "Xamarin.TensorFlow.Lite.API",
```
Comment (jpobst) [inline config.json:1623]: `API` -> `Api`

File: config.json [hunk 3]
```diff
@@ -1998,6 +2007,22 @@
         "nugetVersion": "116.3.0",
         "nugetId": "Xamarin.Firebase.Analytics",
         "dependencyOnly": true
+      },
+      {
+        "groupId": "com.google.errorprone",
```
Comment (jpobst) [inline config.json:2012]: I suspect these `dependencyOnly` entries won't be needed if the bound versions aren't reverted to earlier versions.

Comment (moljac) [reply config.json:2012]: There are 4 parallel PRs being worked on GPS, FB, MLKit and dependencies

Some of those could be from other PRs and I did not clean up `dependencyOnle = true`, but in this case this is false assumption:

```
Dependency errors : 3
1
	System.Exception: 
Artifact
	io.grpc.grpc-api:1.43.2
has unknown 'Runtime' dependency 
	com.google.errorprone.error_prone_annotations:2.9.0
Either fulfill or exclude this dependency.

2
	System.Exception: 
Artifact
	io.grpc.grpc-core:1.43.2
has unknown 'Runtime' dependency 
	com.google.errorprone.error_prone_annotations:2.9.0
Either fulfill or exclude this dependency.

3
	System.Exception: 
Artifact
	io.grpc.grpc-stub:1.43.2
has unknown 'Runtime' dependency 
	com.google.errorprone.error_prone_annotations:2.9.0
Either fulfill or exclude this dependency.
```

File: source/org.chromium.net/cronet-embedded/Transforms/Metadata.Namespaces.xml [hunk 4]
```diff
@@ -112,5 +112,11 @@
         >
         Xamarin.Chromium.Native_Test
     </attr>    
+    <attr
+        path="/api/package[@name='org.chromium.base.lifetime']"
+        name="managedName"
+        >
+        Xamarin.Chromium.Base.LifeTime
```
[the remark this answers is not in the document]
Comment (moljac) [reply source/org.chromium.net/cronet-embedded/Transforms/Metadata.Namespaces.xml]: fixed

**Pull request closed without merging**